app/routes/_landing.($lang).docs/hooks/useDocumentationSearch.tsx

```diff
@@ -63,13 +63,14 @@ export const useDocumentationSearch = ({
     return searchResults.length > 0
       ? searchResults
           .filter((doc) => doc?.translations && doc.translations.length > 0)
-          .map((doc, i) => ({
+          .map((doc) => ({
             ...doc,
             link: docsLinks.find(({ id }) => id === doc.id)?.link,
             value:
               doc && doc.translations && doc.translations[0]?.title
                 ? doc?.translations[0]?.title
                 : doc.id,
+            description: doc && doc.translations && doc.translations[0]?.summary,
           }))
```

Comment:
Why is this needed?

Reply:
To show a summary/description of each search result inside the dropdown

Reply:
I though it wasn't working because I didn't see anything showing so I took the time to do this and add data in directus. I'll remove it, but I do think it's worth while adding dummy data for this kind of thing to not cause confusion....

Reply:
Anyways It's been removed